Makefile.work

```diff
@@ -283,6 +283,7 @@ showtag:
 
 init :
 	@git submodule update --init --recursive
+	@git submodule foreach --recursive git remote update &>/dev/null
```

Comment:
Maybe we should also add this to `make reset` below?

Reply:
possibly, I did see make reset didn't cleanup everything once. we could try this on that too.

Reply:
Just to clarify @jleveque, are you requesting a change to be added to this PR or suggesting experiment with make reset separately?

Reply:
@yxieca: Either way is fine. If you want to test it and add it you can, otherwise we can take note for a future PR.

Reply:
Thanks Joe, per our discussion offline and my test results. I moved the recursive "git remote update" to make reset and placed in front of --init.